src/interfaces/userInterface.ts

```diff
@@ -0,0 +1,33 @@
+export interface IUser {
+    customer_key: string,
+    phone: string,
+    gender: string,
+    orientation: string,
+    location: {
+        type: string,
```

Comment:
what is the use of type?

Reply:
Actually, the geojson is of the form  { type : 'Point', coordinates : [lat, long]}. That's why it must be there. the 'type' field also can be of the lineString, polygon. etc.